### PR TITLE
Implemented Social Share with Shorten URL

### PR DESCRIPTION
**Pull request by hereisnaman**
screen shot 2017-10-13 at 11 04 25 pm

### CODE REVIEW

**Comment by hereisnaman** [comment]
@championswimmer @abhishek97 try live at https://namankumar.me/ide/

**Comment by abhishek97** [comment]
@hereisnaman first of all, fix your commit history

File: src/components/editor/shortcuts.vue [hunk 1]
```diff
@@ -43,76 +43,4 @@
 </script>
 
 <style scoped>
-
-
-  .modal-mask {
```
Comment (abhishek97) [inline src/components/editor/shortcuts.vue]: Why did you move these?

Comment (hereisnaman) [reply src/components/editor/shortcuts.vue]: The same css is used for modal in shortcuts component as well as share component and may get used in other components too included in menuBar component, so I moved it to menuBar component.
